documents/invoice.md

```diff
@@ -127,58 +129,163 @@ instance.invoices.fetch(invoiceId)
 **Response:**
 ```json
 {
-  "id": "inv_E7q0tqkxBRzdau",
+    "amount": 1000,
+    "amount_due": 1000,
+    "amount_paid": 0,
+    "billing_end": null,
+    "billing_start": null,
+    "cancelled_at": null,
+    "comment": null,
+    "created_at": 1653596202,
+    "currency": "INR",
+    "currency_symbol": "₹",
+    "customer_details": {
+        "billing_address": null,
+        "contact": "99991111999",
+        "customer_contact": "99991111999",
+        "customer_email": "gauraa.kumar@example.com",
+        "customer_name": "Gauravss Kumar",
+        "email": "gauraa.kumar@example.com",
+        "gstin": null,
+        "id": "cust_JDdNazagOgg9Ig",
+        "name": "Gauravss Kumar",
+        "shipping_address": null
+    },
+    "customer_id": "cust_JDdNazagOgg9Ig",
+    "date": 1589994898,
+    "description": null,
+    "email_status": "sent",
+    "entity": "invoice",
+    "expire_by": null,
+    "expired_at": null,
+    "first_payment_min_amount": null,
+    "gross_amount": 1000,
+    "group_taxes_discounts": false,
+    "id": "inv_JZz7g9hSZS9IsG",
+    "idempotency_key": null,
+    "invoice_number": null,
+    "issued_at": 1653596202,
+    "line_items": [
+        {
+            "amount": 1000,
+            "currency": "INR",
+            "description": null,
+            "gross_amount": 1000,
+            "hsn_code": null,
+            "id": "li_JZz7gAI2W3Arg2",
+            "item_id": "item_J7lZCyxMVeEtYB",
+            "name": "Test item",
+            "net_amount": 1000,
+            "quantity": 1,
+            "ref_id": null,
+            "ref_type": null,
+            "sac_code": null,
+            "tax_amount": 0,
+            "tax_inclusive": false,
+            "tax_rate": null,
+            "taxable_amount": 1000,
+            "taxes": [],
+            "type": "invoice",
+            "unit": null,
+            "unit_amount": 1000
+        }
+    ],
+    "notes": [],
+    "order_id": "order_JZz7gBTZjtUgBO",
+    "paid_at": null,
+    "partial_payment": false,
+    "payment_id": null,
+    "receipt": null,
+    "reminder_status": null,
+    "short_url": "https://rzp.io/i/DGpanoT",
+    "sms_status": "pending",
+    "status": "issued",
+    "subscription_status": null,
+    "supply_state_code": null,
+    "tax_amount": 0,
+    "taxable_amount": 1000,
+    "terms": null,
+    "type": "invoice",
+    "user_id": null,
+    "view_less": true
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Update invoice
+
+```js
+instance.invoices.edit(invoiceId,{
+  "line_items": [
+    {
+      "id": "li_DAweOizsysoJU6",
+      "name": "Book / English August - Updated name and quantity",
+      "quantity": 1
+    },
+    {
+      "name": "Book / A Wild Sheep Chase",
+      "amount": 200,
+      "currency": "INR",
+      "quantity": 1
+    }
+  ],
+  "notes": {
+    "updated-key": "An updated note."
+  }
+})
+```
```

Comment:
404 error : The id provided does not exist 
please check

Reply:
try this from my credentials
`instance.invoices.edit("inv_JiBytUxRvmsHkH",{
  "line_items": [
    {
      "id": "li_JiC7jAgrAIqTI9",
      "name": "Book / English August - Updated name and quantity",
      "quantity": 1
    },
    {
      "name": "Book / A Wild Sheep Chase",
      "amount": 200,
      "currency": "INR",
      "quantity": 1
    }
  ],
  "notes": {
    "updated-key": "An updated note."
  }
})`